service/controller/app/v1/resource/chartoperator/resource.go

```diff
@@ -103,7 +103,7 @@ func (r Resource) installChartOperator(ctx context.Context, cr v1alpha1.App) err
 		if apierrors.IsNotFound(err) {
 			r.logger.LogCtx(ctx, "level", "debug", "message", "can't find chart-operator app CR")
 			r.logger.LogCtx(ctx, "level", "debug", "message", "cancelling the resource")
-			resourcecanceledcontext.SetCanceled(ctx)
+			reconciliationcanceledcontext.SetCanceled(ctx)
```

Comment:
Without these we can't install chart-operator so there is no point continuing with other resources.